plugins/modules/configuration_item_info.py

```diff
@@ -256,13 +263,14 @@ def main():
                 "sys_id",
                 "query",
                 "configuration_item_mapping",
+                "sysparm_query",
                 "sysparm_display_value",
             ),
             sys_class_name=dict(
                 type="str",
             ),
         ),
-        mutually_exclusive=[("sys_id", "query")],
+        mutually_exclusive=[("sys_id", "query"), ("sysparm_query", "query")],
```

Comment:
```suggestion
        mutually_exclusive=[("sys_id", "query"), ("sysparm_query", "query"), ("sys"id", "sysparm_query")],
```

Reply:
Add `sys_id` and `sysparm_query` as mutually exclusive in configuration_item_info.py.